connectors/cli.py

```diff
@@ -86,6 +97,12 @@ def _parser():
 
 
 async def _start_service(config, loop):
+    """Starts the service.
+
+    Steps:
+    - performs a preflight checks using `PreflightCheck`
+    - instanciates a `MultiService` instance and run its `run` async function
```

Comment:
```suggestion
    - instantiates a `MultiService` instance and runs its `run` async function
```

Reply:
I'm not sure about the description here:
- should it explicitly explain what services it runs?
- should it mention the signal handling?

Reply:
```suggestion
    - starts a series of services via a `MultiService` instance
```